backend/resources/dynamodb.yml

```diff
@@ -0,0 +1,16 @@
+Resources:
+  DojoServerlessTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: dojo-serverless-table
+      AttributeDefinitions:
+        - AttributeName: partitionKey
+          AttributeType: S
+        - AttributeName: rangeKey
```

Comment:
ils appellent pas ça `sortKey` dans les docs ?

Reply:
Yes you're right